e2e/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+
+services:
+  e2e-tests:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    platform: linux/arm64 # Use ARM64 platform for Apple Silicon Macs
```

Comment:
I assume this is needed because of the chromium driver binary, but can we parameterize it, so we can run on linux/amd64 as well? Maybe we can have a `PLATFORM` in the Makefile which defaults to `linux/arm64`, and is passed here, so you can run `make test.ui.docker PLATFORM=linux/amd64`